__tests__/hooks/useGroupInfo.test.js

```diff
@@ -1,7 +1,7 @@
 import * as launchGallery from 'react-native-image-picker';
 import React from 'react';
 import {Alert} from 'react-native';
-import {act, renderHook} from '@testing-library/react-hooks';
+import {act, cleanup, renderHook} from '@testing-library/react-hooks';
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `cleanup` function from `@testing-library/react-hooks` is imported but it's not used in the old hunk. It's good to see that it's now being used in the `afterEach` hook in the new hunk. This ensures that after each test, if the component is unmounted, any remaining tasks that would run during this phase are cancelled, improving the reliability of tests.
```diff
- import {act, renderHook} from '@testing-library/react-hooks';
+ import {act, cleanup, renderHook} from '@testing-library/react-hooks';
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->